teamfoundationcontrol.cpp

```diff
@@ -105,14 +109,8 @@ bool TeamFoundationControl::managesDirectory(const QString &directory, QString *
     if (!isManaged)
         return false;
 
-    QDir dir(directory);
-    while (true)
-    {
-        QTC_ASSERT(topLevel, return true);
-        *topLevel = dir.absolutePath();
-        if (!dir.cdUp() || !managesDirectoryEx(dir.absolutePath()))
-            break;
-    }
+    QTC_ASSERT(topLevel, return true);

```

Comment:
I am not sure I understand this - is it because we only care about project commands now and not the entire repository that we don't have to walk up until we find the repository path? What about files in subdirectories, would they not still have to walk up to project root?